resources/public/api/definition.json

```diff
@@ -9,7 +9,7 @@
   "api": {
     "name": "IV Orchestration",
     "description": "This service checks if users are authorised and allowed to retrieve session data",
-    "context": "iv-orchestration",
+    "context": "individuals",
```

Comment:
The context needs to be unique so this needs to be "individuals/iv-orchestration"